Dockerfile

```diff
@@ -14,11 +14,29 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+FROM registry.access.redhat.com/ubi9/ubi-minimal as downloads
```

Comment:
At some point, I'm hoping that Red Hat starts packaging cosign as its own image. This will allow us to use that image directly (hopefully) in the task instead of having to install cosign here.

Reply:
We could pin to an appstudio-utils version, but this seems like the safest plan.

Reply:
I definitely don't want to use an appstudio image here.